<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>SliderWindow</class>
 <widget class="QWidget" name="SliderWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Slider</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Slider 1</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="2" >
       <widget class="QLabel" name="max0" >
        <property name="text" >
         <string>100</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="min0" >
        <property name="text" >
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3" >
       <widget class="QSlider" name="slider0" >
        <property name="minimumSize" >
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="cursor" >
         <cursor>13</cursor>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition" >
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval" >
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="value1" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>45</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>0</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="title" >
      <string>Slider 2</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="max1" >
        <property name="text" >
         <string>100</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="min1" >
        <property name="text" >
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3" >
       <widget class="QSlider" name="slider1" >
        <property name="minimumSize" >
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="cursor" >
         <cursor>13</cursor>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition" >
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval" >
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="value1_2" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>45</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>0</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Slider 3</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" colspan="3" >
       <widget class="QSlider" name="slider2" >
        <property name="minimumSize" >
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="cursor" >
         <cursor>13</cursor>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition" >
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval" >
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="max2" >
        <property name="text" >
         <string>100</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="min2" >
        <property name="text" >
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="value1_3" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>45</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>0</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Slider 4</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="2" >
       <widget class="QLabel" name="max3" >
        <property name="text" >
         <string>100</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="min3" >
        <property name="text" >
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3" >
       <widget class="QSlider" name="slider3" >
        <property name="minimumSize" >
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="cursor" >
         <cursor>13</cursor>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition" >
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval" >
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="value1_4" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>45</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>0</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>21</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>slider0</sender>
   <signal>valueChanged(int)</signal>
   <receiver>value1</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>231</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel" >
     <x>322</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>value1_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>259</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel" >
     <x>315</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>value1_3</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>267</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel" >
     <x>309</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>value1_4</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>262</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel" >
     <x>306</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
